documentation/sequence diagrams/authentication/User Registration Diagramm.txt

```diff
@@ -0,0 +1,41 @@
+title User Registration Model
+
+participant User
+participant UI component
+participant Authentication/views
+participant Validators
+participant Models
+participant User email
+participant DataBase
+
+User->+UI component: input data
+User->UI component: click 'register' button
+UI component->+Authentication/views: registration(post request)

```

Comment:
there is no return RESPONSE_400_EMPTY_JSON , RESPONSE_400_EXISTED_EMAIL

Reply:
done